### ISSUE TITLE
bower package?

### ISSUE THREAD

**Issue by YonathanB** [classlinkinc/angular-material-time-picker#20]
Hi,
Why don't you have a bower for it?
Standard is to use **npm** for _server-side library_ & **bower** for _front_. Since it's a front functionality, I think it would be better to manage it via bower

**Comment by leonn** [comment]
@YonathanB  it is registered in bower, but does not contain the main bower file and main setup, so you must set it in your bower file

```
"overrides": {
    "angular-material-time-picker":{
      "main": [
        "dist/md-time-picker.css",
        "dist/md-time-picker.js"
      ]
    }
```

the correct fix is if someone add a bower file

**Comment by Andreasbaaserud** [comment]
This is included in release 1.0.8, out now.
